tests/kernel/pipe/pipe_api/src/test_pipe_contexts.c

```diff
@@ -189,27 +301,85 @@ void test_pipe_user_thread2thread(void)
 }
 #endif
 
+/**
+ * @brief Test resource pool free
+ * @see z_mem_pool_malloc()
```

Comment:
we do not have z_mem_pool_malloc in the tree.

Reply:
Replaced the comment by k_heap_alloc.